src/main/java/stringcalculator/CalculatorNumbers.java

```diff
@@ -0,0 +1,27 @@
+package stringcalculator;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class CalculatorNumbers {
+    public static final String DEFAULT_DELIMETER = ":|,";
+    private List<Integer> calculatorNumbers;
```

Comment:
`calculatorNumbers` 상태가 한번 초기화된 뒤, 변경될 일이 없다면 재할당할 수 없도록 `final`로 지정해주셔도 좋을 것 같아요. 😃

```suggestion
    private final List<Integer> calculatorNumbers;
```

Reply:
감사합니다. 불변처리는 생각이 미흡했네요 반영 완료했습니다!